plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeUpdatablePageSource.java

```diff
@@ -558,7 +556,6 @@ private DataFileInfo copyParquetPageSource(DeltaLakeWriter fileWriter)
             if (successfulWrite) {
```

Comment:
Do note that the code will behave slightly differently after this change.

https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

> Note: A try-with-resources statement can have catch and finally blocks just like an ordinary try statement. In a try-with-resources statement, any catch or finally block is run after the resources declared have been closed.

As I understand it, `fileWriter.commit()` statement will be executed after `connectorPageSource.close()`

Reply:
Yes, Thanks for catching this. I will revert this change.